common/scripted_effects/wc_scripted_effects.txt

```diff
@@ -4480,4 +4480,76 @@ break_free_from_from_invader_effect = {
 		}
 		switch_to_random_non_evil_religion_effect = yes
 	}
+}
+
+unleash_an_old_god_effect = {
+	if = {
+		limit = { has_character_flag = freeing_of_cthun_flag }
+		1613 = { save_event_target_as = target_province }
+	}
+	if = {
+		limit = { has_character_flag = freeing_of_nzoth_flag }
+		2201 = { save_event_target_as = target_province }
+	}
+	if = {
+		limit = { has_character_flag = freeing_of_yogg_saron_flag }
+		858 = { save_event_target_as = target_province }
+	}
+	
+	if = {
+		limit = {
+			character = ROOT
+			cthun_offmap_decisions_enabled_trigger = yes
```

Comment:
That's very strange. First, you use the flag to define who wakes up, then, you use the trigger.